### ISSUE TITLE
Bug: Screen Reader NVDA - reading all dates after focused date

### ISSUE THREAD

**Issue by sandeep-chennupati** [gpbl/react-day-picker#1471]
### Bug description

When the ‘DatePicker’ modal expands with NVDA, focus is given to the current date of the month, but all the dates after this date are announced in one message. After all the dates have been announced, and using the arrow keys, focus has moved to the last date of the month which is the last one read in the message. Focus should have remained on the current date of the month but it doesn’t.

### To reproduce

Open https://react-day-picker.js.org/guides/input-fields with NVDA on. And click on the date picker icon on the right side of the input field.

### Expected behavior

When the ‘Date Picker’ modal expands, focus is received on the current date of the current month. Using the swipe method or arrow keys, focus is given to ‘tomorrows’ day/date’ within the calendar.

**Addition Notes**: 
Current behavior is similar to https://www.w3.org/TR/2019/NOTE-wai-aria-practices-1.1-20190814/examples/dialog-modal/datepicker-dialog.html
But the latest version of W3 has the right behavior : https://w3c.github.io/aria-practices/examples/dialog-modal/datepicker-dialog.html.
The difference between them is the old version is having buttons inside td whereas the latest version is using just the td.

**Comment by sandeep-chennupati** [comment]
**Expected behavior:** 
On NVDA -> When the ‘Date Picker’ modal expands, focus is received on the current date of the current month. Using the swipe method or arrow keys, focus is given to ‘tomorrows’ day/date’ within the calendar.

This is not happening on NVDA.
